docs/en/proximity-flow.rst

```diff
@@ -198,6 +198,11 @@ Each mdoc Request MUST be compliant with the following structure, and MUST inclu
 
        This component MUST be present only if `ReaderAuthAllSupport` is set to `true` in the DeviceEngagement structure, and individual `readerAuth` fields are not used.
 
+.. note::
+    **Requesting the Wallet Attestation**
+
+    The Relying Party which requests a Wallet Attestation MUST do so by adding an object in the **docRequest** array having only the ``docType`` set to ``it.wallet.trust-registry.WalletAttestation.1``, it MUST NOT include the ``nameSpaces`` parameter in the request as the Wallet Attestation is not a Digital Credential but a proof of the Wallet Instance's trustworthiness and capabilities.
```

Comment:
'it.wallet.trust-registry.WalletAttestation.1' is just an example. It should be regulated that the reverse trust anchor domain must be used.